src/Ledger/Introduction.lagda

```diff
@@ -35,7 +38,7 @@ explanations may be missing or wrong.
 
 \begin{NoConway}
 \begin{center}
-Repository: \url{https://github.com/IntersectMBO/formal-ledger-specifications}
+Repository: \url{\repourl}
```

Comment:
in case the repo address changes in future (again)

Reply:
:+1: